src/item/ItemComponentsTrait.php

```diff
@@ -47,6 +57,14 @@ public function hasComponent(string $name): bool {
 	}
 
 	public function getComponents(): CompoundTag {
+
+	    $armorSlotEnchantable =  [
+            0 => 'armor_helmet',
+            1 => 'armor_torso',
+            2 => 'armor_legs',
+            3 => 'armor_feet'
+        ];
```

Comment:
why is this defined so early as it's only used in one case ? it could also be replaced with a match case

Reply:
Btw i doubt about the "armor_torso"

Reply:
Use " instead of '